upload/system/library/collivery.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+class collivery
```

Comment:
Unless it interferes with autoloading in OpenCart - please use `PascalCase` for all class names